src/base/static/components/organisms/site-header.js

```diff
@@ -292,33 +294,47 @@ const LogoTitleWrapper = styled(props => (
 }));
 
 const navItemMappings = {
+  // eslint-disable-next-line react/display-name
   internal_link: linkProps => (
     <NavLink
       height="24px"
       position={linkProps.position}
       href={linkProps.navBarItem.url}
+      ariaLabel={`navigate to: ${linkProps.navBarItem.title}`}
       onClick={linkProps.onClick}
     >
       {linkProps.children}
     </NavLink>
   ),
+  // eslint-disable-next-line react/display-name
   left_sidebar_toggle: linkProps => (
-    <NavButtonWrapper position={linkProps.position}>
-      <NavButton
-        color={"tertiary"}
-        onClick={() => {
-          linkProps.onClick();
-          linkProps.setLeftSidebarComponent(linkProps.navBarItem.component);
-          linkProps.setLeftSidebarExpanded(!linkProps.isLeftSidebarExpanded);
-        }}
-      >
-        {linkProps.children}
-      </NavButton>
-    </NavButtonWrapper>
+    <NavButton
+      css={theme => ({
+        [mq[1]]: {
+          alignItems: "center",
+          borderLeft:
```

Comment:
@jalMogo -- note that moving the border here makes it take on the border radius of the `NavButton`, giving the button dividers in the header a curve to them.

Is it problematic from an accessibility standpoint to restore the `NavButtonWrapper`s to address this issue?